src/ThreeJsPanel.ts

```diff
@@ -372,6 +379,23 @@ export class ThreeJsPanel {
     this.updateOrthoScaleBarVisibility();
   }
 
+  setOrthoScaleBarPosition(marginX: number, marginY: number, corner: ViewportCorner) {
+    const { style } = this.orthoScaleBarElement;
+
+    style.removeProperty("top");
+    style.removeProperty("bottom");
+    style.removeProperty("left");
+    style.removeProperty("right");
+
+    const xProp = corner % 2 ? "right" : "left";
+    const yProp = corner % 4 < 2 ? "top" : "bottom";
```

Comment:
This `%2` and `%4` stuff is a bit clunky and would probably be better as a function that lives closer to the enum itself (can TS enums have member functions?).. or else make it much less dependent on the count and order of the enum values